libs/klv/src/main/java/org/codice/alliance/libs/klv/FrameCenterKlvProcessor.java

```diff
@@ -26,15 +31,23 @@
 /**
  * Uses {@link Stanag4609TransportStreamParser#FRAME_CENTER_LATITUDE} and
  * {@link Stanag4609TransportStreamParser#FRAME_CENTER_LONGITUDE} to generate a WKT LINESTRING
- * and store it in the metacard attribute {@link AttributeNameConstants#FRAME_CENTER}.
+ * and store it in the metacard attribute {@link AttributeNameConstants#FRAME_CENTER}. Callers must
+ * supply a {@link Configuration} that contains a postive (&gt;0) Integer for
+ * {@link Configuration#SUBSAMPLE_COUNT}.
  */
-public class FrameCenterKlvProcessor extends MultipleFieldKlvProcessor {
+public class FrameCenterKlvProcessor implements KlvProcessor {
+
+    public static final Integer MIN_SUBSAMPLE_COUNT = 1;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FrameCenterKlvProcessor.class);
 
     /**
      * The name of the metacard attribute being set.
      */
     private static final String ATTRIBUTE_NAME = AttributeNameConstants.FRAME_CENTER;
 
+    private final List<String> stanagFieldNames;
```

Comment:
You aren't leaking this out of the class and only set it once in the ctor. It would probably be clearer if you made it completely immutable as a Guava `ImmutableList`.